services/user.js

```diff
@@ -794,7 +794,8 @@ class UserService {
     } 
     async getCollaboratorsByStudyID(studyID, submitterID) {
         const query = {
-            "studies": {"$in": [studyID, "All"]}, "role": USER.ROLES.SUBMITTER, _id: {"$ne": submitterID}
+            "role": USER.ROLES.SUBMITTER, _id: {"$ne": submitterID},
+            "$or": [{"studies": {"$in": [studyID, "All"]}}, {"studies._id": {"$in": [studyID, "All"]}}]
```

Comment:
Are we supposed to store only study ids for the studies? This might lead some confusion.

Reply:
In user collection,  the studies property is array of object {_id: String} or array of string (backward compatible).